### PR TITLE
docs: readme cleanup

### PR DESCRIPTION
**Pull request by CalebBurton**
- replace non-ASCII punctuation with ASCII equivalent (e.g. `“` --> `"`)
- typo fix: schems --> schema
- re-add documentation for the `random` module, which was mistakenly removed in 11116da0962ff4683d1c69c2c2d1e9aed3801bfe (_all_ of the methods were removed instead of only the deprecated methods)
- clarify that the `random.boolean()` and `random.image()` methods are deprecated 
- <s>change contribution examples to use pnpm since that's now required as of a273f67066c7c19b01ff4348eb9d7e857615cca6</s> addressed by #235
- <s>add `pnpm build` to the examples, since otherwise they will throw an error</s> addressed by #235

### CODE REVIEW

**Comment by Shinigami92** [comment]
I will reevaluate this after #155 was merged
So there will be some conflicts and you need to rebase when that is merged

**Comment by CalebBurton** [comment]
Oh, good! I hadn't seen that, I'll rebase off that branch and work at resolving the conflicts proactively

**Comment by Shinigami92** [comment]
You can now rebase

**Review by Shinigami92 — DISMISSED** [review]

**Review by import-brain — APPROVED** [review]

**Comment by CalebBurton** [comment]
Rebased again to resolve new merge conflicts

File: src/random.ts [hunk 1]
```diff
@@ -233,8 +234,12 @@ export class Random {
    * locale
    *
    * @method faker.random.image
+   * @deprecated
    */
   image() {
+    console.log(
+      'Deprecation Warning: faker.random.image is now located in faker.image.image'
```
Comment (Shinigami92) [inline src/random.ts:241]: We have a test for the random module to check if this message get's logged out.
You could add a test-case for this function there.

Comment (CalebBurton) [reply src/random.ts:241]: Will do 👍

**Review by Shinigami92 — APPROVED** [review]

**Review by import-brain — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**